app/errors.py

```diff
@@ -0,0 +1,13 @@
+class VaccineError(Exception):
+    pass

```

Comment:
Consider adding a constructor to the `VaccineError` class to accept a custom error message. This will allow derived classes to provide more context-specific messages.